_locales/en/messages.json

```diff
@@ -1503,6 +1524,59 @@
         "message": "Signal Strength"
     },
 
+    "motorsNumber1":{
+        "message": "Motor - 1"
+    },
+    "motorsNumber2":{
+        "message": "Motor - 2"
+    },
+    "motorsNumber3":{
+        "message": "Motor - 3"
+    },
+    "motorsNumber4":{
+        "message": "Motor - 4"
+    },
+    "motorsNumber5":{
+        "message": "Motor - 5"
+    },
+    "motorsNumber6":{
+        "message": "Motor - 6"
+    },
+    "motorsNumber7":{
+        "message": "Motor - 7"
+    },
+    "motorsNumber8":{
+        "message": "Motor - 8"
+    },
+
+    "servosNumber1":{
+        "message": "servo - 1"
```

Comment:
i think this should be Servo not servo, as it was befor ;)
also i would prever servoNumber3 and motorNumber3 over servosNumber3 and motorsNumber3

Reply:
No, in the "original code" it was 'servo - 1':

https://github.com/betaflight/betaflight-configurator/blob/5b5290c8ebb76591ea1ca8b450c182e884e0ed2b/tabs/motors.html#L130

I will change the motors and servos to the singular form.

Reply:
True, I will change it.